app/main.py

```diff
@@ -305,57 +305,113 @@ def cardinal(converted_data, path):
+        getVisited(): boolean, returns current state of visited
+        getCost(): int, returns cost attribute of the tile
+    """
+    #isFood = False
 
-    May need logical update for snake bodies but it should be fine
+    def __init__(self, xCoord, yCoord, pathCost):
+        self.x = xCoord
+        self.y = yCoord
+        self.visited = False
+        self.cost = pathCost
+    
+    def getCoord(self):
+        return [self.x, self.y]
+    
+    def getX(self):
+        return self.x
 
-    Args:
-        converted_data (json): python readable json
-        path (list): path from a*
+    def getY(self):
+        return self.y
 
-    Return:
-        updated board (list) with potentially new values around smaller snakes
-        heads
+    def visit(self):
+        self.visited = True
+
+    def getVisited(self):
+        return self.visited
+    
+    def getCost(self):
+        return cost
+
+
+def whatDoYourSnakeEyesSee(pathBoard, xPos, yPos):
     """
-    me = converted_data["you"]["id"]
-    board = pathBoard
-    # other snakes heads will be assigned xy
-    for z in converted_data["board"]["snakes"]:
-        name = z["id"]
-        for a in z["body"]:
-            if ((str(name) != str(me)) and
-                (len(z["body"]) < len(converted_data["you"]["body"]))):
-                if (a == z["body"][0]):
-                    x = a['x']
-                    y = a['y']
-                    if(x < converted_data['board']['width'] and x > 0):
-                        if(pathBoard[y][x+1] != -1):
-                            board[y][x+1] = 3
-                        if(pathBoard[y][x-1] != -1):
-                            board[y][x-1] = 3
-                    if(y < converted_data['board']['width'] and y > 0):
-                        if(pathBoard[y+1][x] != -1):
-                            board[y+1][x] = 3
-                        if(pathBoard[y-1][x] != -1):
-                            board[y-1][x] = 3
+    This method finds the tiles it is possible to path to from any tiles coordinates, and the tiles that form walls around/in this area,
+    Then creates lists for this information.
+    Args:
+        pathBoard (array): integer representation of current board 
+        xPos(int): the x coordinate of the tile we want to search from
+        yPos(int): the y coordinate of the tile we want to search from
+    OBJECT_ORIENTED_TODO: pass in a board object and tile object instead
+    Returns:
+        an list of two lists, the first contains all the Tile objects it is possible and viable to path to, the second 
+        contains all Tile objects that form the walls. Can compare these lists to lists of all food, for example, to get
+        just food we can path too.
+    OBJECT_ORIENTED_TODO: return board objects instead eventually maybe
+    """
+    board_width = len(pathBoard[0])
+    board_height = len(pathBoard)
+    allBoardTiles = [[None for x in range(board_width)] for y in range(board_height)] 
```

Comment:
Please review variable name to reflect python naming conventions (all_board_tiles)